### PR TITLE
Add work and about me sections

### PR DESCRIPTION
**Pull request by tajemouti**
1.  Implemented the HTML CSS linters into the project repo;
2. Added index.html and style.css files to the project;
3. Added a Header for the portfolio page;
4. Added a Headline section;
5. Edited README.md file;
6. Edited LICENSE.md file;
7. Added a Work section;
8. Added  an "About me" section;
9. Re-edited index.html and style.css files;
10. Added a demo live to the README.md file;
11. Executed the requested changes;
12. Fixed linters errors except for Webhint.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,48 +7,203 @@
...
-              <img src="./svg/medium.svg" alt="Medium">
+              <img src="default/SVG/IconMedium.svg" alt="Medium">
             </a>
           </li>
         </ul>
       </section>
+      <section class="work-section">
+        <span>
+          <h2 class="work-section-title">My Recent Works</h2>
+          <hr class="divider">
```
Comment (franclobo) [inline index.html:55]: - [ ] Please 🙏🏽 use the correct tag for the best practices of HTML & CSS, use `div` instead of `hx`

**Review by franclobo — APPROVED** [review]
Hi @tajemouti, 

# Required Changes ♻️

istockphoto-1079018918-612x612

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Good UI ✔️ 
- Best practices of HTML & CSS ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tajemouti) [reply index.html:55]: Thank you for your advice.. I appreciated..

**Review by Ofarouq310 — APPROVED** [review]
Hi @tajemouti, ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇  

giphy

## Highlights
✅ Followed GitHub flow.
✅ Descriptive PR title & summary.
✅ Meaningful commit messages.

## Optional suggestions

- I believe there is an element on your project that is causing a horizontal bar, which conflicts with the responsiveness your application should have, kindly address that issue. 
  Screenshot 2023-06-10 205438

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**